README.md

```diff
@@ -111,6 +111,12 @@ user named `daemon` defined within that file-system.
             "source": "sysfs",
             "destination": "/sys",
             "options": "nosuid,noexec,nodev"
+        },
+        {
+            "type": "cgroup",
```

Comment:
I don't know, this looks like  pretty confusing, because actually we ignoring "source".